TW 841 Cycle 9 edits (#4530)
postmanlabs/postman-docs: src/pages/docs/getting-started/postman-account.md

> After you sign in, you will be redirected back to the Postman desktop app.

Reviewer: On this page, I think there is an additional variation of "log in/out" that needs to be changed on line: `201`

If you're on a Professional, Basic, or Free plan, you can change the email address associated with your Postman account. Open your [account settings](https://go.postman.co/settings/me/account/) page. Select your avatar in the Postman header, then select **Settings > Account**. Edit your email address and select **Update Email** to save changes.

If you're on an Enterprise plan, you must contact your Team Admins to update the email address associated with your Postman account.

### Resetting your password

If you're on a Professional, Basic, or Free plan, you can change your password if you're already signed in by navigating to your [account settings](https://go.postman.co/settings/me/account/) page. Select your avatar in the Postman header, then select **Settings > Account**. Select **Change Password**.

If you aren't signed in to your Postman account, you can recover your username or reset your password from the [**Sign In**](https://identity.getpostman.com/login) page. Select **Trouble signing in?**. Enter your account email, select the **I'm not a robot** checkbox, and select **Submit**. Postman will send you an email with instructions for recovering access to your account.

> > If you still have trouble logging into your Postman account, contact [Postman support](https://support.postman.com/hc/en-us).

Reviewer: ```suggestion
> If you still have trouble signing in to your Postman account, contact [Postman support](https://support.postman.com/hc/en-us).
```

## Managing your active sessions

> You can manage your [active Postman sessions](https://go.postman.co/settings/me/sessions/) by selecting your avatar in the Postman header, then selecting **Active Sessions**. You can view all of the devices you're logged in to Postman on and revoke any sessions you no longer use or recognize. You can also choose to **Revoke all sessions**.

Reviewer: ```suggestion
You can manage your [active Postman sessions](https://go.postman.co/settings/me/sessions/) by selecting your avatar in the Postman header, then selecting **Active Sessions**. You can view all of the devices you're signed in to Postman on and revoke any sessions you no longer use or recognize. You can also choose to **Revoke all sessions**.
```
